dev_tools/tests/data/NXhdf5_validator_1.nxdl.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl" ?>
+<definition xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
+    xmlns="http://definition.nexusformat.org/nxdl/3.1"
+ name="NXhdf5_validator_1"
+ extends="NXobject"
+ type="group"
+ category="application"
+>
+    <doc>This is a dummy NXDL to test out the dataconverter.</doc>
```

Comment:
This should not mention the dataconverter, which is a pynxtools concept. Just say it is a dummy for testing validation.